include/rocksdb/utilities/transaction_db.h

```diff
@@ -203,27 +204,29 @@ class TransactionDB : public StackableDB {
                      const std::vector<ColumnFamilyDescriptor>& column_families,
                      std::vector<ColumnFamilyHandle*>* handles,
                      TransactionDB** dbptr);
-  // The following functions are used to open a TransactionDB internally using
-  // an opened DB or StackableDB.
-  // 1. Call prepareWrap(), passing an empty std::vector<size_t> to
```

Comment:
Seems these comments are still useful to explain how to use PrepareWrap() and Wrap*DB().

Reply:
I did not find them useful. A simple grep would should where this function is used. Listing them in comment does not help understand better what the function does.